src/OrbitGl/ThreadTrack.cpp

```diff
@@ -252,7 +252,12 @@ Color ThreadTrack::GetTimerColor(const TimerInfo& timer_info, bool is_selected,
 
   constexpr uint8_t kOddAlpha = 210;
   if ((timer_info.depth() & 0x1) == 0) {
-    color[3] = kOddAlpha;
+    // The usage of alpha values with possible intersection of timers may lead to a strange visual
+    // behavior where users doesn't understand why some timers "are lighter". Therefore, we
+    // "compute" the odd-alpha manually.
```

Comment:
What you're doing here isn't exactly the same behavior as before. IMO it doesn't really matter because the end result looks good, but I find the comment confusing: Before, alpha-blending was configured to use "source_color * alpha + dest_color * (1 - alpha)" (where source_color is the color in this function, and dest_color the color that was drawn underneath, so the track background in our case). This is usually the "standard" way of configuring alpha-blending.
 
I would simply rename the variable to something like `kOddRowColorMultiplier` and explain in the comment that this is done to have slightly alternating row colors without any mention of alpha.

Reply:
Agree with everything.

- It's not the same (but TBH I didn't have a better idea)
- It's similar and looks good.
- The comment should be rephrased.